lms/templates/instructor/instructor_dashboard_2/executive_summary.html

```diff
@@ -0,0 +1,114 @@
+<%! from django.utils.translation import ugettext as _ %>
+<!DOCTYPE HTML>
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<title>Edx Executive Summary</title>
+<style type="text/css">
+body {
+	font-family: Arial, Helvetica, sans-serif;
+	font-size:14px;
+	line-height:22px;
+	margin: 10px;
+}
+.box-bg {
+	background:#f1f1f1;
+	padding:10px;
+}
+th {
+	padding:5px;
+	background:#ccc;
+}
+h2 {
+	margin-top:0
+}
+</style>
+</head>
+
+<body>
+
+<table width="650" border="0" cellspacing="5" cellpadding="5">
+	<tr>
+		<td align="left" valign="top"><img src="https://d37djvu3ytnwxt.cloudfront.net/static/mit-prof-ed/images/MITPE-Digital-Programs-Logo.png" alt="logo" /></td>
+	</tr>
+	<tr>
+		<td align="left" valign="top" class="box-bg"><h2>${_("Executive Summary")}</h2>
+			<table width="100%">
+				<tr>
+					<td width="300">${_("Total Enrollments:")}</td>
+					<td align="right">${total_enrollments}</td>
+				</tr>
+				<tr>
+					<td>${_("Gross Revenue Collected to date:")}</td>
+					<td align="right">${gross_revenue}</td>
+				</tr>
+				<tr>
+					<td>${_("Gross Pending Revenue:")}</td>
+					<td align="right">${gross_pending_revenue}</td>
+				</tr>
+				<tr>
+					<td>${_("Total Enrollments Refunded:")}</td>
+					<td align="right">${total_enrollments_refunded}</td>
+				</tr>
+				<tr>
+					<td>${_("Total Amount Refunded:")}</td>
+					<td align="right">${total_amount_refunded}</td>
+				</tr>
+				<tr>
+					<td>${_("Average price paid:")}</td>
+					<td align="right">${average_paid_price}</td>
+				</tr>
+			</table></td>
```

Comment:
One thing I'm seeing on the sand box is not rounding to two decimal places, e.g. $33.33333333333. Not sure if the rounding should happen here in the template or in the code.